### PR TITLE
[XPU] add swiglu xpu test

### PR DESCRIPTION
**Pull request by HarperCy**
### PR Category
Operator Mechanism

### PR Types
New features

### Description
add swiglu xpu test

### CODE REVIEW

**Review by RuohengMa — APPROVED** [review]
LGTM

**Review by tizhou86 — APPROVED** [review]
LGTM

File: paddle/phi/kernels/xpu/swiglu_kernel_grad.cc [hunk 1]
```diff
@@ -64,7 +64,7 @@ void SwiGluGradKernel(const Context& ctx,
                              reinterpret_cast<XPUType*>(dx_data),
                              dims_vec,
                              axis,
-                             false,
```
Comment (QingshuChen) [inline paddle/phi/kernels/xpu/swiglu_kernel_grad.cc:67]: 这个参数是什么意思?直接改?

**Review by QingshuChen — APPROVED** [review]

File: paddle/phi/kernels/xpu/swiglu_kernel_grad.cc (hunk 1 above)
Comment (HarperCy) [reply paddle/phi/kernels/xpu/swiglu_kernel_grad.cc:67]: false对右半silu，true对左半silu

**Pull request merged**